tests/tests.sh

```diff
@@ -1,16 +1,19 @@
 #!/usr/bin/env bash
 set -euxo pipefail
 
-COMPOSE_PROJECT_NAME=autoheal-test
+COMPOSE_PROJECT_NAME=${1:-autoheal-test}
+export COMPOSE_PROJECT_NAME
 
 function cleanup()
 {
     exit_status=$?
     echo "exit was $exit_status"
-    docker-compose -f docker-compose.autoheal.yml -f docker-compose.yml rm -f || true
+    # stop autoheal first, to stop it restarting the test containers while we try to stop them
+    docker-compose -p "$COMPOSE_PROJECT_NAME" stop autoheal
```

Comment:
I think we might not need `-p "$COMPOSE_PROJECT_NAME"` as per line no 5 its already part of environment which docker-compose supports reading from.

Reply:
the `-p "$COMPOSE_PROJECT_NAME"` creates a different project if you have 2 parallel runs, this enables our CI to run 2 tests for autoheal (e.g. upstream and our custom one) at the same time without sharing container names. (one is prefixed with autoheal_tests_12345_ the other one with autoheal_tests_12346_ and both do not interact with each other)

the export is just for the watch-autoheal so it knows which project to listen to.

Reply:
gotcha, docker-compose also reads `COMPOSE_PROJECT_NAME` from environment, only pointing no need to explicitly pass it if it's already part of your environment

Reply:
Thanks for the explanation, didn't know that. For others (or me in the future) here is the reference for that: https://docs.docker.com/compose/reference/envvars/#compose_project_name